android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileFragment.kt

```diff
@@ -83,14 +91,40 @@ class ProfileFragment : Fragment() {
     return ComposeView(requireContext()).apply {
       setViewCompositionStrategy(ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed)
       setContent {
-        AppTheme {
-          ProfileScreen(
-            navController = findNavController(),
-            profileUiState = profileViewModel.profileUiState.value,
-            onEvent = profileViewModel::onEvent,
-            snackStateFlow = profileViewModel.snackBarStateFlow,
+        val appConfig = appMainViewModel.applicationConfiguration
+        val scaffoldState = rememberScaffoldState()
+
+        LaunchedEffect(Unit) {
+          profileViewModel.snackBarStateFlow.hookSnackBar(
+              scaffoldState = scaffoldState,
+              resourceData = null,
+              navController = findNavController(),
           )
         }
+
+        AppTheme {
+          Scaffold(
```

Comment:
The profile screen already has a scaffold. I do not think this is needed/necessary. The ProfileViewModel already includes a snackbar flow.

Reply:
Perhaps the issue to investigate here is why the Snackbar is not appearing upon submission of a Questionnaire.

Reply:
Yes, I just realized that while investigating. I've stopped adding code and I'm still exploring the flow. I apologize for that